doc/api/net.md

```diff
@@ -73,7 +73,7 @@ var server = net.createServer((socket) => {
 
 // grab a random port.
 server.listen(() => {
-  address = server.address();
+  var address = server.address();
   console.log('opened server on %j', address);
```

Comment:
@Trott that sadly resulted in `opened server on [object Object]`, so went with what @jasnell suggested.